internal/db/event_logs.go

```diff
@@ -9,9 +9,11 @@ import (
 	"github.com/keegancsmith/sqlf"
 	"github.com/lib/pq"
 	"github.com/pkg/errors"
+
 	"github.com/sourcegraph/sourcegraph/cmd/frontend/types"
 	"github.com/sourcegraph/sourcegraph/internal/db/dbconn"
 	"github.com/sourcegraph/sourcegraph/internal/db/dbutil"
+	intSecrets "github.com/sourcegraph/sourcegraph/internal/secrets"
```

Comment:
To be consistent with other places, we should name it like this?
```suggestion
	secretsPkg "github.com/sourcegraph/sourcegraph/internal/secrets"
```